Sources/Sentry/SentryClient.m

```diff
@@ -585,6 +584,30 @@ - (void)captureUserFeedback:(SentryUserFeedback *)userFeedback
     [self.transportAdapter sendUserFeedback:userFeedback];
 }
 
+- (void)captureFeedback:(SentryFeedback *)feedback withScope:(SentryScope *)scope
```

Comment:
`m`: It would be great to add some test(s) to `SentryClientTests` for this new method.

Reply:
What is the value of all the tests that only validate whether a method was called, using Invocations? if i write a test that validates the contents of the envelope, it implicitly validates this took place. I assume they have provided unique value to you in the past, I just can't imagine what it is so would appreciate any explanation.

Reply:
beforeSend is a good candidate for this. I think envelope content validation can check for stacktraces, screenshots, and viewHierarchy in addition to the core things like the feedback message, name and email.

Reply:
the hooks actually still aren't implemented, so i will defer testing those to a later PR.